script/fetchneedles

```diff
@@ -51,6 +51,8 @@ git_update() {
     branch="${1:-"master"}"
     git gc --auto --quiet
     git fetch -q origin
+    # Clear any uncommitted changes that would prevent a rebase
+    git reset --hard HEAD
```

Comment:
I would feel safer if we introduce this together with a parameter `--force` or environment variables "force". Can you do that?

Reply:
Sounds good. I added a check for `force=1` which defaults to `0` and if the rebase fails the option is suggested as a way to handle it automatically.